frontend/src/components/management/WorkflowAssignment.tsx

```diff
@@ -29,33 +30,60 @@ export function WorkflowAssignment({ fixedFilter }: LabworkSamplesProps) {
     const wholeFilters = useMemo(() => ({ ...filters, ...fixedFilters }), [filters, fixedFilters])
 
     const samplesTableCallbacks = usePagedItemsActionsCallbacks(SamplesTableActions)
+    const [searchParams] = useSearchParams()
     useEffect(() => {
+        samplesTableCallbacks.clearFixedFiltersCallback()
```

Comment:
isn't the return callback dealing with that already ? or this is to prevent cross-effect from regular sample table fixed filters ?

Reply:
It's to prevent cross-effect from regular sample table fixed filters.